Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'bzip2-ruby', :github => 'chewi/bzip2-ruby', :branch => 'master'
```

Comment:
Is there something unreleased we need from this master branch? Seems like some unofficial fork. If there's some memleak issue, how was it solved by scaptimony?